worker/src/router.rs

```diff
@@ -1,7 +1,7 @@
 use std::{collections::HashMap, future::Future, rc::Rc};
 
 use futures_util::future::LocalBoxFuture;
-use matchit::{Match, Node};
+use matchit::{Match, Router as Node};
```

Comment:
`Node` has been renamed in https://github.com/ibraheemdev/matchit/releases/tag/v0.5.0
I had to perform this rename since `Router` conflicts with the struct defined in line 45